env/GAEA.env

```
@@ -0,0 +1,308 @@
+#! /usr/bin/env bash
+
+if [[ $# -ne 1 ]]; then
+
+    echo "Must specify an input argument to set runtime environment variables!"
+    echo "argument can be any one of the following:"
+    echo "atmanlrun atmensanlrun aeroanlrun snowanl"
+    echo "anal sfcanl fcst post metp"
+    echo "eobs eupd ecen efcs epos"
+    echo "postsnd awips gempak"
```

```suggestion
    echo "argument can be any one of the following:"
```
This is incomplete list, so might as well remove all of them.